runtime/parachains/src/disputes.rs

```diff
@@ -1001,6 +1027,10 @@ impl<T: Config> Pallet<T> {
 						// by the duplicate checks above. It's only the first one which
 						// may not already have been filtered out.
 						filter.remove_index(first_index_in_set);
+
+						// Removing individual statments can cause the dispute to become onesided.
+						// Checking that (again) is done after the loop. Remove the bit indices.
+						summary.new_flags.set(validator_index.0 as _, false);
```

Comment:
Why is this suddenly necessary?

Reply:
See my next comment, it's not used outside the scope of this `fn` so it's alright to modify it at this point.

Reply:
Now, that I check this more closely, it makes even less sense. `new_flags` has type: [DisputeStateFlags](https://github.com/paritytech/polkadot/blob/31a254b9174388422f9fe94a831c90a64f4c172c/runtime/parachains/src/disputes.rs#L498), I don't understand why we would use `ValidatorIndex` as index here. :confused:

Reply:
Yeah, you're correct. That makes no sense at all.